locales/en.yml

```
@@ -34,4 +34,4 @@ en:
       link_text_color: "The text color of links within the banner."
       link_text_color_dark: "The text color of links within the banner when system dark mode is detected. <br><br><br><h2>Banner Content Customization</h2> Customize the content of the banner with HTML. If you need to style your custom content with CSS, create a separate theme component. Leave the content blank to exclude a column. While a fourth column is provided, it will likely result in too much content for the available space. Be sure you are happy with your configuration across different brower widths."
       fourth_column_content: '<br><br><h2>Column Sizes</h2>Use a percentage to make columns more wide or narrow. When set to "auto", the column will fill the available space evenly with other "auto" columns.'
-      fourth_column_size: '<br><br><h2>Column Icons</h2>Use font awesome or a link to an image for the icon at the top of the columns. <a href="https://fontawesome.com/v5/search?o=r&m=free" target="_blank">Click Here</a> to view the list of available icons.'
+      fourth_column_size: '<br><br><h2>Column Icons</h2>Use font awesome or a link to an image for the icon at the top of the columns. <a href="https://fontawesome.com/search?o=r&m=free" target="_blank">Click Here</a> to view the list of available icons.'
```

version-prefixed URL to be safe
```suggestion
      fourth_column_size: '<br><br><h2>Column Icons</h2>Use font awesome or a link to an image for the icon at the top of the columns. <a href="https://fontawesome.com/v6/search?o=r&m=free" target="_blank">Click Here</a> to view the list of available icons.'
```